frontend/src/components/browse-sightings-page/BrowseSightings.tsx

```diff
@@ -17,11 +20,23 @@ export const BrowseSightings: React.FC = () => {
   return (
     <>
       <h1>Reported Sightings</h1>
-      <ul>
-        {sightings.map((sighting, index) => (
-          <SightingCard sighting={sighting} key={sighting.id} />
-        ))}
-      </ul>
+      <button
+        onClick={() => {
+          setMapToggle(!mapToggle);
+        }}
+      >
+        Sightings Map
+      </button>
+      {!mapToggle ? (
+        <ul>
+          {sightings.map((sighting, index) => (
+            <SightingCard sighting={sighting} key={sighting.id} />
+          ))}
+        </ul>
+      ) : (
+        // <h1>HELLO !!!!! </h1>
+        <MapPage sightingList={sightings} />
+      )}
```

Comment:
Remove the "`!`" and flip the ternary operator:
```suggestion
      {mapToggle ? (
        // <h1>HELLO !!!!! </h1>
        <MapPage sightingList={sightings} />
      ) : (
        <ul>
          {sightings.map((sighting, index) => (
            <SightingCard sighting={sighting} key={sighting.id} />
          ))}
        </ul>
      )}
```